### PR TITLE
Teaching page | spring 2020

### PR DESCRIPTION
**Pull request by julbinb**

### CODE REVIEW

File: teaching.rkt [hunk 1]
```diff
@@ -193,6 +198,7 @@
                  }
                  @li{
                      @strong{4400} Programming Languages @br{}
+                     @indent[] S 2020 @a[href: "https://felleisen.org/matthias/4400-s20/" target: "_blank"]{link} @instructor{Matthias Felleisen} @br{}
```
Comment (mfelleisen) [inline teaching.rkt:201]: Apparently the course is called Principles of Programming Languages.

Comment (julbinb) [reply teaching.rkt:201]: Hm, on Registrar it's called Programming Languages
Screenshot_20200502_103134_org mozilla firefox

Comment (mfelleisen) [reply teaching.rkt:201]: OK. (Everyone refers to this thing as PPL and the PhD level as iPPL.)

Comment (julbinb) [reply teaching.rkt:201]: Maybe somebody could make a name change request! :)

**Pull request merged**